code/game/objects/items/weapons/material/gravemarker.dm

```diff
@@ -3,33 +3,35 @@
 	desc = "An object used in marking graves."
 	icon_state = "gravemarker"
 	w_class = ITEMSIZE_LARGE
-	fragile = 1
-	force_divisor = 0.65
-	thrown_force_divisor = 0.25
+	material_significance = MATERIAL_SIGNIFICANCE_WEAPON_MEDIUM
 
 	var/icon_changes = 1	//Does the sprite change when you put words on it?
 	var/grave_name = ""		//Name of the intended occupant
 	var/epitaph = ""		//A quick little blurb
 
 /obj/item/material/gravemarker/attackby(obj/item/W, mob/user as mob)
 	if(W.is_screwdriver())
+		var/datum/material/material = get_primary_material()
+		var/time_mult = (material.hardness > 0)? material.hardness / 100 : 1 / (material.hardness / 100)
 		var/carving_1 = sanitizeSafe(input(user, "Who is \the [src.name] for?", "Gravestone Naming", null)  as text, MAX_NAME_LEN)
 		if(carving_1)
 			user.visible_message("[user] starts carving \the [src.name].", "You start carving \the [src.name].")
-			if(do_after(user, material.hardness * W.tool_speed))
+			if(do_after(user, time_mult * 1 SECONDS * W.tool_speed))
 				user.visible_message("[user] carves something into \the [src.name].", "You carve your message into \the [src.name].")
 				grave_name += carving_1
 				update_icon()
 		var/carving_2 = sanitizeSafe(input(user, "What message should \the [src.name] have?", "Epitaph Carving", null)  as text, MAX_NAME_LEN)
 		if(carving_2)
 			user.visible_message("[user] starts carving \the [src.name].", "You start carving \the [src.name].")
-			if(do_after(user, material.hardness * W.tool_speed))
+			if(do_after(user, time_mult * 1 SECONDS * W.tool_speed))
 				user.visible_message("[user] carves something into \the [src.name].", "You carve your message into \the [src.name].")
 				epitaph += carving_2
 				update_icon()
 	if(W.is_wrench())
+		var/datum/material/material = get_primary_material()
+		var/time_mult = (material.hardness > 0)? material.hardness / 100 : 1 / (material.hardness / 100)
 		user.visible_message("[user] starts carving \the [src.name].", "You start carving \the [src.name].")
-		if(do_after(user, material.hardness * W.tool_speed))
+		if(do_after(user, time_mult * 1 SECONDS * W.tool_speed))
```

Comment:
SECONDS is a multiplier, they should be `time_mult SECONDS`

Reply:
this is actually to signify '1 second times multiplier times tool speed', aka 1 second is base